index.html

```diff
@@ -5,11 +5,964 @@
   <meta charset="utf-8">
   <title>TNW</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="build/css/reset.css">
-  <link rel="stylesheet" href="build/css/styles.css">
+  <link rel="stylesheet" href="src/css/reset.css">
+  <link rel="stylesheet" href="src/css/styles.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@700&display=swap" rel="stylesheet">
 </head>
 
 <body>
-</body>
+  <header>
+    <div class="medium-logo">
+      <img src="assets/images/tnw.svg" alt="Company logo" width="60">
+    </div>
+    <nav class="nav1">
+      <div class="top-links1">
+        <a href="#" class="color-white">News</a>
+        <a href="#" class="color-grey">Events</a>
+        <a href="#" class="color-grey">Business</a>
+        <a href="#" class="color-grey">AMAs</a>
+        <a href="#" class="color-grey">Spaces</a>
+        <a href="#" class="color-grey">Terms & Conditions</a>
+      </div>
+      <div class="top-links2">
+        <a href="#" class="color-grey">About</a>
+        <a href="#" class="color-grey">Advertise</a>
+        <a href="#" class="color-grey">Jobs</a>
+        <a href="#" class="color-grey">Contact</a>
+      </div>
+    </nav>
 
-</html>
+    <nav class="nav2">
+      <div class="top-links3">
+        <a>LATEST</a>
+        <a>HARD FORK</a>
+        <a>PLUGGED</a>
+        <a>README</a>
+        <a>GROWTH QUARTERS</a>
+      </div>
+      <div>
+        <a href="#" style="margin-right: 15px;"><i class="fa fa-search"></i></a>
+      </div>
+    </nav>
+
+    <nav class="nav3">
+      <div class="nav-logo">
+        <a class="nav-logo" href="https://thenextweb.com">
+          <img class="nav-logo" alt="The Next Web" src="./assets/images/tnw.svg">
+        </a>
+      </div>
+      <div class="nav-icons">
+        <div class="nav-ics display-flex">
+          <a class="nav-ics nav-ics-sch" href="#"><i class="fa fa-search"></i></a>
+        </div>
+        <div class="nav-ics display-flex">
+          <a class="nav-ics nav-ics-menu" href="#"><i class="fa fa-bars"></i></a>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section class="story-section">
+      <article class="cover-story">
+        <div class="cover-bgi" style="background-image: url('./assets/images/lucid-air-configurator.jpg');">
```

Comment:
Dear reviewer, please note that the inline style is an image. I do consider it absurd to create a class whose sole content is a static URL to an asset. If the change is really, really needed, please make sure to suggest a solution because, again, I do think it is absurd to make a class for a static asset URL that is the background.